zerver/tests/test_signup.py

```diff
@@ -733,9 +733,19 @@ def test_invite_user_as_admin_from_normal_account(self) -> None:
         """
         self.login(self.example_email('hamlet'))
         invitee = self.nonreg_email('alice')
-        response = self.invite(invitee, ["Denmark"], invite_as_admin="true")
+        response = self.invite(invitee, ["Denmark"], invite_as=2)
```

Comment:
This should use the PreregistrationUser.INVITE_AS['REALM_ADMIN'], not 2 -- more readable that way.